patientsearch/src/js/components/Utility.js

```diff
@@ -1,9 +1,14 @@
-export function sendRequest (url) {
+export function sendRequest (url, params) {
+    params = params || {};
     // Return a new promise.
     return new Promise(function(resolve, reject) {
       // Do the usual XHR stuff
       var req = new XMLHttpRequest();
       req.open('GET', url);
+      if (params.nocache) {
+        // via Cache-Control header:
+        req.setRequestHeader("Cache-Control", "no-cache, no-store, max-age=0");
```

Comment:
If this is the same caching issue we've fixed in flask before, I think you'll only need `no-cache`
```suggestion
        req.setRequestHeader("Cache-Control", "no-cache");
```